aws/ci_build

```diff
@@ -104,6 +109,7 @@ def main()
 
       HipChat.log message, color: 'red'
       HipChat.log "/quote #{log}", color: 'gray', message_format: 'text'
+      HipChat.log "/quote #{status}", color: 'gray', message_format: 'text'
```

Comment:
the `HipChat` class sends messages to both HipChat and Slack in its current implementation. If there's a TODO, it should be bulk-renaming the HipChat class to something more generic like `ChatLog` or something (and out of the scope of this particular fix).